### DISCUSSION TITLE
How to use the same Duration Predictor for multiple speakers?

### DISCUSSION THREAD

**Discussion by acbrun0** [coqui-ai/TTS#2334] [General Q&A]
Hello,

As I understand it, conditioning the DP on the speaker embedding means that each speaker will use different weights on the DP. Furthermore, setting `condition_dp_on_speaker=false` on the configuration file would cause the DP to average out between all speakers.

I am satisfied with the duration prediction results on one of the speakers. How would I go about making it so that all speakers use the same DP as that one?

What comes to mind is training with only that speaker, setting `freeze_DP=true` and `condition_dp_on_speaker=false`, and adding the rest of the speakers. However, this would mean going back quite a while in the training process. Is there another way?

Thank you in advance.

**Comment by huongtxvega** [comment]
I have the same question
